docker-compose.yml

```
@@ -0,0 +1,19 @@
+version: "3.7"
+
+services:
+  nginx:
+    image: nginx:latest
+    ports:
+      - ${NGINX_PORT:-8000}:80
```

du coup si tu optes pour le port 80 par défaut plutôt que 8000, peut être tu faire la même chose ici ?
```suggestion
      - ${NGINX_PORT:-80}:80
```